### PR TITLE
Fix GS compile error by removing inline asm

### PR DESCRIPTION
**Pull request by theresa-m**
Signed-off-by: Theresa Mammarella <Theresa.T.Mammarella@ibm.com>

### CODE REVIEW

File: port/win32/omrintrospect.c [hunk 1]
```diff
@@ -307,46 +307,9 @@ setup_native_thread(J9ThreadWalkState *state, CONTEXT *sigContext)
 		 * current thread. It seems to work, however, if we populate the EIP and EBP context
 		 * data manually (which is what the assembler does below).
 		 */
-#ifdef WIN64
```
Comment (keithc-ca) [inline port/win32/omrintrospect.c]: Perhaps the comment above should be updated given the removal of the assembly code?

Comment (theresa-m) [reply port/win32/omrintrospect.c]: Thanks :) updated.

**Comment by charliegracie** [comment]
No need for further CI testing as this is in Windows only code.

**Comment by DanHeidinga** [comment]
Did anyone confirm that the comment regarding the limitations from MSDB was no longer correct?

**Comment by charliegracie** [comment]
We removed the usage of the call that the comment was talking about.  In the past we used the removed API on 32 bit since the API we are using for both 32/64 bit now was not available.  The API used now was made available on XP as far as I can see we should be ok.

**Comment by DanHeidinga** [comment]
Reading https://bugs.chromium.org/p/crashpad/issues/detail?id=53 , which was updated in 2015, the function is broken on 32bit windows.  The claim is that it returns the caller's caller's values.

**Comment by keithc-ca** [comment]
The one or two frames that might be omitted from that context aren't helpful; they'd be setup_native_thread and startDo/nextDo. The interesting frames are much deeper in the stack.

**Comment by DanHeidinga** [comment]
Seriously?  You'd be happy working with an inconsistent set of registers?

**Comment by keithc-ca** [comment]
The old assembly version was only capturing Eip and Ebp; I'm assuming RtlCaptureContext at least captures them from the same frame.

**Comment by DanHeidinga** [comment]
Did you read the link I pasted?

**Comment by DanHeidinga** [comment]
Also, the original code did two things: `GetThreadContext` to lookup the context and then the assembly code to fix up the `ebp` & `eip`

**Comment by DanHeidinga** [comment]
I've opened an OpenJ9 issue to validate that OpenJ9's javacores, which use this code to fetch the native stack, are still correct once this change hits the OpenJ9 builds.

**Pull request merged**